### PR TITLE
[978] Edit Classification and Qualification

### PR DESCRIPTION
**Pull request by Sybrand**

### CODE REVIEW

**Comment by Sybrand** [comment]
[EPIC#Registry | Edit Classification and Qualification |5| !!! [gherkin]](https://trello.com/c/ngOsfdje/978-epicregistry-edit-classification-and-qualification-5-gherkin)

**Comment by Sybrand** [comment]
[EPIC#Registry | Edit Classification and Qualification |5| !!! [gherkin]](https://trello.com/c/ngOsfdje/978-epicregistry-edit-classification-and-qualification-5-gherkin)

File: app/frontend/src/registry/components/people/ApplicationDetail.vue [hunk 1]
```diff
@@ -197,4 +306,7 @@ export default {
 .registry-subtle {
   font-size: 0.9rem;
 }
+.fixed-width .custom-control-label {
+  width: 220px;
+}
 </style>
```
Comment (stephenhillier) [inline app/frontend/src/registry/components/people/ApplicationDetail.vue]: I tried everything out on dev, works good.

File: app/frontend/src/registry/components/people/ApplicationDetail.vue [hunk 2]
```diff
@@ -129,52 +179,114 @@ export default {
...
+          this.$store.commit(SET_ERROR, error.response)
+        })
+    },
+    focusCancelModal () {
+      // focus the "cancel" button in the confirm discard popup
+      this.$refs.cancelSubmitCancelBtn.focus()
+    },
+    saveApplication () {
+      this.loading = true
+      const copy = Object.assign(this.applicationFormValue)
```
[the remark this answers is not in the document]
Comment (Sybrand) [reply app/frontend/src/registry/components/people/ApplicationDetail.vue]: Full points for spotting that!

**Pull request merged**